README.md

```diff
@@ -1 +1,23 @@
-# portfoliofull
\ No newline at end of file
+# Portfolio Mobile Version
+
+This is a skeleton of a mobile version portfolio design this is showing my portfolio template and all my recent projects
+
+## Technologies used
+
+HTML, CSS
+
+### Install
+
+Download the project from github and unzip the file and enjoy
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: https://github.com/Danelif
+  -Twitter :https://twitter.com/DanElfemuk
+  Linkedin : https://www.linkedin.com/in/dan-elifeleti-5811a3106/
+
```

Comment:
Kindly customise the handles of your social and professional networks. Use the [microverse template](https://github.com/microverseinc/readme-template/blob/master/README.md).